### PR TITLE
PRs 6503, 6320, 6324

### PR DESCRIPTION
**Pull request by peiying2**
1). PR 6503: add new file 
2). PR 6320: foundation legal updates
3). PR 6324: Foundation ownership of m.o. site.

### CODE REVIEW

File: en-US/foundation/index.lang [hunk 1]
```diff
@@ -14,10 +15,15 @@ The nonprofit Mozilla Foundation believes the Internet must always remain a glob
 The direct work of the Mozilla Foundation focuses on fueling the movement for an open Internet. We do this by connecting open Internet leaders with each other and by mobilizing grassroots activists around the world.
 
 
+## TAG: foundation_legal_201810
```
Comment (flodolo) [inline en-US/foundation/index.lang]: Why did you put the tag on this string, when you're adding another one?

**Comment by flodolo** [comment]
> Having said that, there are plenty of hard-coded strings in the template.

Actually, that might not be the case :-\

File: en-US/firefox/accounts-2018.lang [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+;Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.
+Securely sync your passwords, bookmarks and tabs across all your devices. Get a Firefox Account now – One log-in – Power and privacy everywhere.
+
+
+;One log-in. Power and privacy everywhere.
+One log-in. Power and privacy everywhere.
+
+
+# CTA: https://accounts.firefox.com/signup
+Create a Firefox Account
```
Comment (flodolo) [inline en-US/firefox/accounts-2018.lang]: Please look at the Errors page…

**Comment by flodolo** [comment]
Still missing this bit

> You should add the string for the FAQ page, and we can add the tag later if needed.

File: en-US/privacy/faq.lang [hunk 3]
```diff
@@ -155,6 +156,12 @@ What about my account data?
 We are big believers of data minimization and not asking for things we don't need.
 
 
+## TAG: firefox-accounts-112018
+;You don't need an account to use Firefox. <a href="%(accounts)s"accounts }}">Accounts</a> are required to sync data across devices, but we only ask you for an email address. We don't want to know things like your name, address, birthday and phone number.
+You don't need an account to use Firefox. <a href="%(accounts)s">Accounts</a> are required to sync data across devices, but we only ask you for an email address. We don't want to know things like your name, address, birthday and phone number.
```
Comment (flodolo) [inline en-US/privacy/faq.lang]: Have you checked for errors?

Comment (peiying2) [reply en-US/privacy/faq.lang]: Gosh, ;-\. However, on my error page, nothing like this shows up, including the ones you mentioned earlier.

**Pull request merged**